code/modules/clothing/under/ties.dm

```diff
@@ -504,19 +504,41 @@
 	hold = /obj/item/storage/internal/tie/vest
 
 /obj/item/clothing/tie/storage/white_vest
-	name = "white webbing vest"
-	desc = "A clean white Nylon vest with large pockets specially designed for medical supplies"
+	name = "surgical vest"
+	desc = "A clean white Nylon vest with large pockets specially designed for holding surgical supplies."
 	icon_state = "vest_white"
 	hold = /obj/item/storage/internal/tie/white_vest
 
 /obj/item/storage/internal/tie/white_vest
 	storage_slots = 8
 	can_hold = list(
 		/obj/item/tool/surgery, 
-		/obj/item/stack/medical/advanced/bruise_pack, 
-		/obj/item/stack/medical/advanced/ointment, 
+		/obj/item/stack/medical/advanced, 
+		/obj/item/clothing/mask/surgical,
+		/obj/item/clothing/gloves/latex,
 		/obj/item/stack/nanopaste)
 
+/obj/item/storage/internal/tie/white_vest/medic
```

Comment:
should be `/obj/item/clothing/tie/storage/white_vest/medic`

Reply:
Thank you, this was doing my head in all night